osu.Game/Overlays/Profile/Header/Components/GroupBadge.cs

```diff
@@ -68,7 +71,18 @@ private void load(OverlayColourProvider? colourProvider, RulesetStore rulesets)
                         }
                     }
                 }
-            });
+            };
+
+            AddRangeInternal(groupDrawables);
+
+            // Probationary groups have an opacity of 60%
+            if (group.IsProbationary)
+            {
+                foreach (var drawable in groupDrawables)
```

Comment:
Why not just do `Alpha = 0.6f;` on the whole container instead of setting alpha on each of the children?